website/docs/r/sns_topic_subscription.html.markdown

```diff
@@ -242,6 +242,8 @@ The following arguments are supported:
 * `endpoint_auto_confirms` - (Optional) Boolean indicating whether the end point is capable of [auto confirming subscription](http://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.html#SendMessageToHttp.prepare) e.g., PagerDuty (default is false)
 * `confirmation_timeout_in_minutes` - (Optional) Integer indicating number of minutes to wait in retying mode for fetching subscription arn before marking it as failure. Only applicable for http and https protocols (default is 1 minute).
 * `raw_message_delivery` - (Optional) Boolean indicating whether or not to enable raw message delivery (the original message is directly passed, not wrapped in JSON with the original message in the message property) (default is false).
+* `filter_policy` - (Optional) JSON String with the filter policy that will be used in the subscription to filter messages seen by the target resource. Refer to the SNS docs for more details.
```

Comment:
Thank you!!! I totally missed this yesterday. 😢  Can you please also link to the appropriate documentation for these two policies please? e.g. https://docs.aws.amazon.com/sns/latest/dg/message-filtering.html

Reply:
No problem. The delivery policy doc is not specific about the format https://docs.aws.amazon.com/sns/latest/dg/DeliveryPolicies.html . There is another page with an example to set this via CLI but is less specific and the policy is incomplete :) https://docs.aws.amazon.com/sns/latest/dg/json-formats.html